6.bulk_Morphology_Elastic_Network/3.model_coefficients/scripts/1.model_coefficients.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import joblib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+# In[2]:
+
+
+cell_type = "SHSY5Y"
+
+
+# In[3]:
+
+
+# set import paths
+model_path = pathlib.Path(
+    f"../../1.train_models/models/regression/{cell_type}/aggregated_with_nomic/IL-1 beta [NSU]_final__all_nomic.joblib"
+)
+
+# set and create output paths
+output_path = pathlib.Path(f"../results/regression/{cell_type}/")
+output_path.mkdir(parents=True, exist_ok=True)
+
+
+# In[4]:
+
+
+# set path of models
+model_path = pathlib.Path(
+    f"../../1.train_models/models/regression/{cell_type}/aggregated_with_nomic/"
+)
+# get the list of files in the directory
+model_files = list(model_path.glob("*.joblib"))
+for model_file in model_files:
```

Comment:
Consider using the `model_path.glob()` generator directly in the for loop below (instead of `model_files`).

Reply:
Updated and fixed!